lib/generators/wcc/templates/contentful

```diff
@@ -0,0 +1,342 @@
+#!/bin/bash
```

Comment:
this is the migration CLI tool, right? name doesn't communicate that

Reply:
it's kind of the "contentful everything tool" right now - the wrapper around all the various nodejs packages.  I'll clarify on the name.